### ISSUE TITLE
[Bug]: graphical glitch

### ISSUE THREAD

**Issue by srkrishnan1989** [minikin/popover#92]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I am using kde neon OS 6.0.4 (Linux) which is based on  Ubuntu OS ,my 

flutter version is 3.22  , the problem is when I activate ShowPopOver function the graphical glitches happening

The code I have used
```
 ListTile(
        title: Text(text),
        trailing: Builder(
          builder: (context) =>
             IconButton(
                onPressed: () => showPopover(
                  width: 100,height: 100,
                    context: context, 
                    bodyBuilder: (context) => NoteSettings(onEditTap: editOnPressed,onDeleteTap: deleteOnPressed,)),
                icon: const Icon(Icons.more_vert))
          
        ),
      )
```

[Screencast_20240609_130140.webm](https://github.com/minikin/popover/assets/6891619/a953d35a-f3ce-4e93-8fe4-49b65c91300e)

### What operating system are you seeing the problem on?

Linux

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by angelocordero** [comment]
Currently experiencing the same issue

**Comment by minikin** [comment]
@srkrishnan1989  @angelocordero feel free to create a fix.

**Comment by merkator95** [comment]
Experiencing the same on 3.22.2 and resolves on flutter channel master. Seems to be a flutter engine issue possibly related to https://github.com/flutter/engine/pull/50754.

**Comment by srkrishnan1989** [comment]
The problem solved on flutter version 3.24

**Comment by srkrishnan1989** [comment]
The problem solved on flutter version 3.24
